_posts/2023-07-31-paris-dev-sprint.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "scikit-learn 2023 In-person Developer Sprint in Paris, France"
+date: July 31, 2023
+
+categories:
+  - Events
+tags:
+  - Sprints
+  - Community
+
+postauthors:
+  - name: Reshama Shaikh
+    website: https://reshamas.github.io
+    image: reshama_shaikh.jpeg 
+  - name: François Goupil
+    email: francois.goupil@inria.fr
+    website: https://github.com/francoisgoupil
+    image: "francois_goupil.jpeg"
+---
+<div>
+  {% include postauthor.html %}
+</div>
+
+
+In June of 2023 (week of June 19 to 23), the scikit-learn held their first Developers sprint since 2019!  The sprint took place in Paris, France at the Dataiku office. The sprint event was an in-person event only and had 29 participants.
```

Comment:
```suggestion
In June of 2023 (week of June 19 to 23), the scikit-learn held its first Developers sprint since 2019!  The sprint took place in Paris, France at the Dataiku office. The sprint event was an in-person event only and had 29 participants.
```